### ISSUE TITLE
Mardown isn't shown properly

### ISSUE THREAD

**Issue by NikoKrause** [clefebvre/cinnamon-spices#7]
Checklist, Table and Line (=====) is not shown properly on Spices Website,
compare this
https://cinnamon-spices.linuxmint.com/applets/view/281
with this
https://github.com/linuxmint/cinnamon-spices-applets/tree/master/CinnVIIStarkMenu@NikoKrause

**Comment by MrElectroNick** [comment]
Maybe that's because GitHub use their own Markdown render?
I can't find a standard which say that (=====) should be converted into horizontal rule

According to this: http://daringfireball.net/projects/markdown/syntax

HORIZONTAL RULES

You can produce a horizontal rule tag (`<hr />`) by placing three or more hyphens, asterisks, or underscores on a line by themselves. If you wish, you may use spaces between the hyphens or asterisks. Each of the following lines will produce a horizontal rule:

`* * *`

`***`

`*****`

`- - -`

`---------------------------------------`

**Comment by NikoKrause** [comment]
Thanks.
The spices website uses this parser: http://parsedown.org/

It doesn't parse the checklist. Other parsers do not as well. I'll replace it with a normal list.

But the lines on the table are not shown. It's not dramatic either. At least it aligns properly.

So I think those problems are all due to the fact, that GitHub uses a slightly different parser for their website.

**Comment by MrElectroNick** [comment]
Yes. GitHub has its own. Lines in the table, missing left borders for quotes is just a CSS issue which is fixable.

Same as line height of headings and their sized. I'll try to make it look more like in github with CSS changes

**Comment by NikoKrause** [comment]
@MrElectroNick 
The gif here https://cinnamon-spices.linuxmint.com/applets/view/268 is too big.
It's bigger than the screen. Is this also an CSS issue?

**Comment by Odyseus** [comment]
Hello, everybody.

@NikoKrause & @MrElectroNick 

Too large images can be easily fixed by the following code as reported [here](https://github.com/clefebvre/cinnamon-spices/issues/10):

```css
img {
    max-width: 100%;
}
```

**Comment by MrElectroNick** [comment]
@Odyseus @NikoKrause - Thanks guys - update is coming

**Comment by NikoKrause** [comment]
Are markdown `codeblocks` also a CSS issue?
https://cinnamon-spices.linuxmint.com/desklets/view/35
vs
https://github.com/linuxmint/cinnamon-spices-desklets/tree/master/googleCalendar@javahelps.com

**Comment by nyrrad** [comment]
Tables and `codeblocks` fixed. Currently visible on master.

Going to leave this issue open for now to see if there are any others missing
